packages/d/dotnet/package.yml

```diff
@@ -29,49 +31,60 @@ builddeps  :
     - dotnet-sdk
     - dotnet-source-built-artifacts
     - git
-    - jq
     - lldb-devel
 rundeps    :
-    - libicu
+    - dotnet-shared
     - sdk :
         - dotnet
+    - shared :
+        - libicu
```

Comment:
This is added in one commit, and then moved in https://github.com/getsolus/packages/pull/5881/commits/dc4f4a2d97b65beafde2016fe572f983318e3bfa. The updated linter will flag that, as that commit now updates a package without bumping the version. The commit also contains changes in two packages, which is generally not allowed.